au4/src/effects/builtin/general/generalviewmodel.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+* Audacity: A Digital Audio Editor
+*/
+#include "generalviewmodel.h"
+
+#include "libraries/lib-effects/Effect.h"
+
+#include "au3wrap/internal/wxtypes_convert.h"
+
+#include "log.h"
+
+using namespace au::effects;
+
+void GeneralViewModel::init()
+{
+    IF_ASSERT_FAILED(!instanceId().isEmpty()) {
+        return;
+    }
+
+    //! NOTE Temporary solution
+    m_effect = reinterpret_cast<Effect*>(instanceId().toLongLong());
+
+    LOGDA() << au3::wxToSting(m_effect->GetSymbol().Internal());
```

Comment:
Are we planning to fix this `wxToSting` typo sometime or is it already too late? 😄

Reply:
I will fix in a next PR, it's never too late :D